icons/pilot.svg

```diff
@@ -0,0 +1,19 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M10 15v2" />
+  <path d="M14 15v2" />
+  <path d="M8 11H7c-1.7 0-3-1.3-3-3V7c0-2.2 1.8-4 4-4h1c1.7 0 3 1.3 3 3v1c0 2.2-1.8 4-4 4z" />
+  <path d="M17 11h-1c-2.2 0-4-1.8-4-4V6c0-1.7 1.3-3 3-3h1c2.2 0 4 1.8 4 4v1c0 1.7-1.3 3-3 3z" />
+  <path d="M18 18h2a2 2 0 0 0 2-2v-3a2 2 0 0 0-2-2h-2v6a4 4 0 0 1-4 4h-4a4 4 0 0 1-4-4v-6H4a2 2 0 0 0-2 2v3a2 2 0 0 0 2 2h2" />
+  <path d="M5 18v4" />
+  <path d="M19 18v4" />
+</svg>
```

Comment:
Maybe something with small eyes for the sake of cuteness? ![icon (49)](https://github.com/lucide-icons/lucide/assets/25524993/e107df60-98eb-48cd-9286-06d404a97ae3)

Reply:
@jguddas Options…

<img width="210" alt="Artboard 1@2x" src="https://github.com/lucide-icons/lucide/assets/7797479/93bb9bb1-8bd6-48b5-9930-a975c19afe20">
<br/>
<img width="666" alt="Untitled" src="https://github.com/lucide-icons/lucide/assets/7797479/5b886be8-7e7e-475a-ae38-3bf3ecf7ea14">

Reply:
@jguddas Implemented…